<ui version="4.0" >
 <class>DeleteWidgtClass</class>
 <widget class="QWidget" name="DeleteWidgtClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DeleteWidgt</string>
  </property>
  <widget class="QGroupBox" name="groupBox" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>60</y>
     <width>341</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title" >
    <string>人员信息删除</string>
   </property>
   <widget class="QLabel" name="label" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>40</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text" >
     <string>请选择要删除的类型：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>100</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>请选择要删除的编号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numberLineEdit" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>100</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>删除</string>
    </property>
   </widget>
   <widget class="QLabel" name="fenquLabel" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>140</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>请选择要删除的分区</string>
    </property>
   </widget>
   <widget class="QComboBox" name="fenquComboBox" >
    <property name="geometry" >
     <rect>
      <x>180</x>
      <y>135</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="measureGroupBox" >
   <property name="geometry" >
    <rect>
     <x>570</x>
     <y>70</y>
     <width>241</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title" >
    <string>测量信息删除</string>
   </property>
   <widget class="QComboBox" name="measureComboBox" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>90</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>40</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>请选择要删除的测量数据：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="measureDelButton" >
    <property name="geometry" >
     <rect>
      <x>60</x>
      <y>220</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>删除测量数据</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
